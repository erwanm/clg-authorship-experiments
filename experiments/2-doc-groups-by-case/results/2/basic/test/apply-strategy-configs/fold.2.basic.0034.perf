0.341000	0.550000	0.620000
