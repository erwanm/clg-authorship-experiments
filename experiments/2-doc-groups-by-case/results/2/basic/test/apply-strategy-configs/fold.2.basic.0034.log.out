apply-config.sh: applying config '/tmp/expe2/2/process/selected-meta-configs/fold.2.strategy-as-meta.basic.0034.model/strategy-configs/fold.2.basic.0034' to cases './results/2/basic/test/apply-strategy-configs/only-cases.tmp.SVI2KbHh'; results written to './results/2/basic/test/apply-strategy-configs/fold.2.basic.0034'
apply-config.sh: done.
